superset/connectors/sqla/views.py

```diff
@@ -167,6 +170,9 @@ class TableColumnInlineView(CompactCRUDMixin, SupersetModelView):
 
 class SqlMetricInlineView(CompactCRUDMixin, SupersetModelView):
     datamodel = SQLAInterface(models.SqlMetric)
+
+    class_permission_name = "Datasource"
```

Comment:
Same as below, I have "Datasource" permission visible, but the older one still persists.